tenable/ad/checker_option/schema.py

```diff
@@ -0,0 +1,43 @@
+import typing
+from marshmallow import fields, validate, ValidationError, pre_load
+from tenable.ad.base.schema import CamelCaseSchema, camelcase
+
+
+class StagedRule(fields.Field):
+    def _deserialize(
+            self,
+            value: typing.Any,
+            attr: typing.Optional[str],
+            data: typing.Optional[typing.Mapping[str, typing.Any]],
+            **kwargs
+    ):
+        if isinstance(value, str) or isinstance(value, bool):
+            return value
+        else:
+            raise ValidationError('Field should be string or boolean.')
```

Comment:
Instead of focusing on writing a more complex mapping rule, wouldn't it make more sense to simply subclass the Boolean field to force it to return an integer value?  As these all relate to query params, the constructed request will _always_ be a string value in the end.


```suggestion
class BoolInt(fields.Boolean):
    def _serialize(value, attr, obj, **kwargs) -> int:
        return int(value) if value else None
```